en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc

```diff
@@ -1,12 +1,22 @@
 = MuleSoft Composer Release Notes
-:page-revdate: June 28, 2023
+:page-revdate: July 3, 2023
 
 Use MuleSoft Composer to build connected experiences faster. You don't need to wait for development resources to unlock data, integrate apps, and automate business processes that create real customer value.
 
 New versions are automatically available once they are released; you don't need to perform any tasks to move to the new version.
 
 See xref:ms_composer_ms_release_notes_ki.adoc[Known Issues and Limitations for MuleSoft Composer] for a list of known issues and limitations by date.
 
+== July 3, 2023
+
+include::_partials/release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+3]
+
+[%header%autowidth.spread]
+|===
+|Issue Resolution |ID
+|In RPA Connector, the *RPA Process Execution Completed* trigger no longer fails every few seconds with an `UNKNOWN ERROR`. |  W-13697777
```

Comment:
```suggestion
|In RPA Connector, the *RPA Process Execution Completed* trigger no longer fails every few seconds with `UNKNOWN ERROR`. |  W-13697777
```
If you leave the (indef) article you should add a word like message after the ``.